foundations/html_css/css-foundations.md

```diff
@@ -259,6 +259,7 @@ img {
   width: 500px;
 }
 ~~~
+For example, if an image's original size was 500px height and 1000px width, using the above CSS would result in a height of 250px.
```

Comment:
Can you add an empty line between the end of the code block (line 261) and your added verbiage?

Reply:
Added an empty line after the code block